flexmeasures/data/models/time_series.py

```diff
@@ -584,6 +584,12 @@ def __init__(
         source: tb.DBBeliefSource,
         **kwargs,
     ):
+        # get a Sensor instance attached to the database session (input sensor is detached)
+        # check out Issue #683 for more details
+        _sensor = Sensor.query.get(sensor.id)
```

Comment:
Could we check whether the instance is detached instead of querying the database regardless?

Reply:
Good idea! I added the check :D

Reply:
Did you push the new check? I don't see a change.

Reply:
Sorry, I forgot to push it. Just did :smile: